plutus-use-cases/src/Language/PlutusTx/Coordination/Contracts/Escrow.hs

```diff
@@ -240,9 +244,7 @@ redeem escrow = do
     let addr = escrowAddress escrow
         valRange = 
             let itvl = Interval.from 1
-                -- an upper bound that does not include the escrow 
-                -- deadline
-                upperBound = Interval.UpperBound (Interval.Finite $ escrowDeadline escrow) False
+                upperBound = Interval.strictUpperBound (escrowDeadline escrow)
             in itvl { Interval.ivTo = upperBound }
```

Comment:
`Interval { ivFrom = lowerBound 1, ivTo = strictUpperBound (escrowDeadline escrow) }`?

Alternatively, if you add `strictFrom` and `strictTo` as well you could write `from 1 /\ strictTo (escrowDeadline escrow)`. So many options.

Reply:
I found a better solution ;)